packages/quick-edit-extension/src/cfs.ts

```diff
@@ -198,6 +198,7 @@ declare module "*.bin" {
 				create: true,
 				overwrite: true,
 				suppressChannelUpdate: true,
+				readOnly: files.readOnly,
```

Comment:
propably not the case, but just to be sure...to we need to `readOnly: files.readOnly ?? false` here? Or will the editor handle a possible `readOnly: undefined`?

Reply:
The property didn't exist before and was fine so it should handle undefined without issue (and does seem to from my tests)
I didn't modify this object structure, just the loaded one

Reply:
all good. was merely a sanity check than anything else